ui/platform_window/x11/x11_window_manager_ozone.cc

```diff
@@ -10,11 +10,19 @@ namespace ui {
 
 X11WindowManagerOzone::X11WindowManagerOzone() : event_grabber_(None) {}
 
-X11WindowManagerOzone::~X11WindowManagerOzone() {}
+X11WindowManagerOzone::~X11WindowManagerOzone() {
```

Comment:
Ok. I understand the point of Add/Remove/Get methods here, and I agree the manager class as it is is weak - hence we need to make it an actual manager.

My suggestions are:
- would it make sense to change X11WindowManagerOzone to work with X11WindowOzone instead of XID?

- the idea of having a global variable is also valid given that it makes the patch smaller (and rebases easier).